packages/browser/src/core/user/__tests__/index.test.ts

```diff
@@ -871,14 +871,19 @@ describe('store', function () {
   })
 
   describe('#get', function () {
-    it('should not not get an empty record', function () {
-      expect(store.get('abc') === undefined)
```

Comment:
Interesting, I wonder how this test was passing before when we should have been returning a value or null.

Reply:
`expect(store.get('abc') === undefined)` will always pass (because it lacks an assertion). It's basically `expect(true)`

Reply:
Ah of course, that makes sense.